centos-release-rpms-spec/centos-release-openshift-origin310.spec

```diff
@@ -10,7 +10,7 @@ Source1:   LICENSE
 URL:       https://wiki.centos.org/SpecialInterestGroup/PaaS/OpenShift
 Provides:  %{name} = %{version}-%{release}
 # This provides the public key
-Requires:  centos-release-paas-common
+Requires:  centos-release-paas-common, centos-release-ansible26
```

Comment:
Do we want this automatically required for 3.10? What’s the impact for folks who have found their own ansible since this was already released?

Reply:
that is a good question.
i'll take out the new changes and only leave in the file which was already used to build the 3.10 rpm.

I was aiming to fix the issue where folks are using 2.7 even for 3.10 but then

- not sure if they are using our rpm or not
- to late to shut the stable door, the horse has bolted already